torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -740,9 +749,19 @@ def fw_fn(*primals):
     def metadata_fn(*primals):
         return inner_fn(fw_only, primals, use_trace_joint=False)
 
-    args_unwrapped = unwrap_tensor_subclasses(
-        args, is_joint_structure=is_joint_structure
-    )
+    if is_joint_structure:
+        args_unwrapped = (
+            unwrap_tensor_subclasses(
+                args[0], is_runtime=False, append_symints=True, subclass_metas=None
```

Comment:
I know we had a decent amount of back-and-forth on exactly when you need to pass in `append_symints=True/False ` and `subclass_meta=...`.

For each call-site, can you add a comment explaining why we are passing these particular values in?